docs/reference/connector/apis/connector-apis.asciidoc

```diff
@@ -3,17 +3,16 @@
 
 preview::[]
 
-++++
-<titleabbrev>Connector APIs</titleabbrev>
```

Comment:
Why are we getting rid of the leading heading element on the website? Search engines make use of it for ranking, so that when someone types `connector apis` the title (here `<h2>`) helps. 

I see this present for other apis: https://www.elastic.co/guide/en/elasticsearch/reference/master/esql-apis.html

Reply:
Ohh I still see the title in preview, is it some trick you used?? 😮

Reply:
haha no it's just abbreviation for sidebar in case the title is extra long, but it's unnecessary in this instance